nodejs/week1/meal_sharing_website/src/backend/routes/meal.js

```diff
@@ -0,0 +1,34 @@
+//Respond with the json for a random meal (including it's reviews)
+
+const express = require("express");
+const router = express();
```

Comment:
Remove. Same explanation than the previous file. Also export only router function.